graphql_compiler/compiler/emit_sql.py

```diff
@@ -107,6 +108,37 @@ def _find_columns_used_outside_folds(sql_schema_info, ir):
     return used_columns
 
 
+def _find_tagged_parameters(expression_from_filter):
+    """Return True if the expression contains a ContextColumn (i.e. a tagged parameter)."""
```

Comment:
From the docstring of `ContextColumn` it seems that it is used when referencing concrete columns in a table, rather than fields in the GraphQL type corresponding to that table. However, for tagged parameters, we know the tag must have been on a field in the GraphQL type. Do we use `ContextColumn` here because that field access (i.e. `ContextField` type) has already been lowered away in a previous step to a `ContextColumn`, or is there another reason for it?

Maybe I'm being paranoid here. I'm just worried that here we seem to be implicitly assuming that `ContextColumn` has to mean the presence of a tagged parameter. Whether that is true or false right now isn't obvious to me, and I'm even more worried about whether that will remain true or false in the future.

And maybe the answer is just "this is temporary code until we implement this functionality". If that's the case, I'd like to make sure we have a plan on how we'd actually implement it, so that the code doesn't accidentally become permanent :)

Reply:
looking into this now...